fvm_dispatch_helper/src/blake2b.rs

```diff
@@ -0,0 +1,11 @@
+use fvm_dispatch::hash::Hasher;
+use multihash::Code;
+use multihash::MultihashDigest;
+pub struct Blake2bHasher {}
+impl Hasher for Blake2bHasher {
+    fn hash(&self, bytes: &[u8]) -> Vec<u8> {
+        let digest = Code::Blake2b256.digest(bytes);
+        // drop the first 4 bytes of the multihash which identify the hash as blake2b256
```

Comment:
The multihash library is probably too high level for this dependency. Find out what it depends on to implement blake2b and use that instead.